pages/platform/ai/deploy_guide_06_billing_concept/guide.en-gb.md

```diff
@@ -97,15 +96,15 @@ The sizing depends on the selected amount of compute resources, check the detail
 
 This storage space can be used by your Docker image, for local operations.
 
-#### Remote Object storage
+#### Remote Object Storage
 
 When working with remote data, you pay separately for the storage of this data.
-The pricing of object storage is apart from the app pricing.
+The pricing of Object Storage is separate from the app pricing.
 
 ### Pricing examples
 
-For these examples, we will take a pricing as 1,95€ / hour per GPU NVIDIA V100s and 0,03€ / hour per CPU.
-Pricing may vary, please refer to official pricing page.
+For these examples, we will take a pricing of 1,95€ / hour per GPU NVIDIA V100s and 0,03€ / hour per CPU.
```

Comment:
For these examples, we will use a pricing of 1,95€ / hour per GPU NVIDIA V100s and 0,03€ / hour per CPU.